packs/opentelemetry_collector/templates/opentelemetry_collector.nomad.tpl

```diff
@@ -73,6 +82,7 @@ EOH
 
       [[- if $vars.task_services ]]
       [[- range $idx, $service := $vars.task_services ]]
+      [[- if or (not ($vars.traefik_config.enabled)) (and ($vars.traefik_config.enabled) (ne $service.service_port_label "otlphttp") (ne $service.service_port_label "otlp")) ]]
```

Comment:
I think this check overwrites the `otlp` and `otlphttp` services from the [`task_services` variable](https://github.com/hashicorp/nomad-pack-community-registry/blob/main/packs/opentelemetry_collector/variables.hcl#L197-L216). 

Go template starts to get a little weird handling this, but I worked out something that merges the service tags if Traefik is enabled: https://github.com/lgfa29/nomad-pack-community-registry/commit/73d071eec4cd9241329acae229c3e9db243c11fd

What do you think of this approach?

Reply:
Nice! I pushed a commit to your branch with this change then 🙂